docs/tutorials/guides/sounds.mdx

```diff
@@ -0,0 +1,286 @@
+
+```csharp
+public Game()
+{
+	Width = 900;
+	Height = 600;
+}
+```
+
+
+## <span class="color-step">Step 4: Load the content</span>
+
+Now, let's proceed with loading the sound as audio content for playback. Before we delve into the code for loading the
+audio content, it's crucial to add the audio file to the project as content. This step ensures it can be loaded.
+Once that's finished, we can enhance the functionality by adding code into the `OnLoad()` method. This code will enable
+us to load the audio content and create a keyboard input object. By utilizing the keyboard, we'll be able to easily
+control the audio playback.
+
+:::info Supported Audio Formats
+Currently, the supported audio formats are _**.mp3**_ and _**.ogg**_.
+:::
+
+### <span class="color-sub-step">Step 4.1: Get audio content</span>
+
+You can provide whatever audio file you want, but if you don't have one, you can download the original tutorial project
+sound [here](https://github.com/KinsonDigital/VelaptorTutorials/raw/master/Sounds/Content/Sounds/deep-consistency.ogg).
+
+Play it to see if you like it.  😀
+<Audio soundUrl="https://github.com/KinsonDigital/VelaptorTutorials/raw/feature/add-sound-guide/Sounds/Content/Sounds/deep-consistency.ogg"/>
+
+### <span class="color-sub-step">Step 4.2: Setup file as content</span>
+
+We need to add the audio file to the project into a particular directory and then set the file to be copied to the
+build output.  This will be the default location where Velaptor will be looking for audio content.
+
+:::note Refer To Guide
+For more info, refer to the [Adding Content](./adding-content) guide.
+:::
+
+### <span class="color-sub-step">Step 4.3: Write code to load content</span>
+
+Now that we have the audio file added to the project, we can write some code to load the audio content.  Add the
+following code to the `OnLoad()` method to load the sound and create our keyboard input object.
+
+```csharp
+    protected override void OnLoad()
+    {
+        this.keyboard = InputFactory.CreateKeyboard();
+        this.music = ContentLoader.LoadSound("deep-consistency");
+
+        base.OnLoad();
+    }
+```
+
+:::tip Content name
+Remember, the name of the audio file without its extension is the name of the content.  The audio file added previously
+is the name of the file if you downloaded the file from the tutorial project.  If you provided an audio file,
+make sure the content name matches the file name that you decided to use.
+:::
+
+
+## <span class="color-step">Step 5: Displaying sound info</span>
+
+So we have an audio file to play, but we need to know some information such as the current duration of the audio
+if it is playing or paused, and some basic instructions for the user on how to control the audio playback.  Since we are going
+to be using the keyboard to control the sound, it makes sense to tell the user which keyboard keys to use to stop, play, and
+other sound-related operations.
```

Comment:
Reword
```suggestion
So we have an audio file to play, but we need to know some information like the current duration of the audio,
if it's playing or paused, and some basic instructions for the user on how to control the audio playback.  Since we are going
to be using the keyboard to control the sound, it makes sense to tell the user which keyboard keys to use to stop, play, and
other sound-related operations.
```